ui/react-components/components/TimeSelector/TimeSelector.jsx

```diff
@@ -8,15 +8,20 @@ import {injectIntl} from "react-intl";
 
 const TimeSelector = props => {
 
+    const {
+        translationKey, defaultValue, onChange,
+        timeSelectionTranslationKey, timeSelectionDefaultValue, defaultTime
+    } = props;
     return (
         <div className={classNames(timeSelector)}>
             <div>
-                <Label translationKey={props.translationKey}
-                       defaultValue={props.defaultValue}/>
+                <Label translationKey={translationKey}
+                       defaultValue={defaultValue}/>
             </div>
-            <AppointmentTimePicker onChange={props.onChange}
-                                   placeHolderTranslationKey={props.timeSelectionTranslationKey}
-                                   defaultValue={props.timeSelectionDefaultValue}/>
+            <AppointmentTimePicker onChange={onChange}
+                                   placeHolderTranslationKey={timeSelectionTranslationKey}
+                                   defaultValue={timeSelectionDefaultValue} defaultTime={defaultTime}
+            />
```

Comment:
defaultValue and defaultTime?

Reply:
defaultValue for placeholder, defaultTime for setting time

Reply:
Please change it to defaultPlaceHolder

Reply:
changed it to placeholderDefaultMessage